benchmark/src/grpc_helpers.rs

```diff
@@ -127,33 +132,36 @@ pub fn submit_and_wait_txn_requests(
 /// ------------------------------------------------------------ ///
 
 /// Send account state request async with a AC client.
+/// Try to unmarshall only the first ResponseItem in the succeeded response.
+/// Return a tuple consisting of address (as account's identifier), and deserialized response item.
 fn get_account_state_async(
     client: &AdmissionControlClient,
-    address: &AccountAddress,
-) -> Result<impl Future<Item = UpdateToLatestLedgerResponse, Error = failure::Error>> {
-    let requested_item = RequestItem::GetAccountState { address: *address };
+    address: AccountAddress,
+) -> Result<impl Future<Item = (AccountAddress, ResponseItem), Error = failure::Error>> {
```

Comment:
Reply to @0-o-0 : Switched to FutureUnordered after adopting your suggestion to return (address, response_item) in `get_account_state_async`